src/components/Codeblock.tsx

```diff
@@ -6,34 +6,32 @@ import Highlight, {
   PrismTheme,
 } from "prism-react-renderer"
 import Prism from "prism-react-renderer/prism"
-import { Box, BoxProps, Flex, useColorModeValue } from "@chakra-ui/react"
 
 // https://github.com/FormidableLabs/prism-react-renderer/tree/master#custom-language-support
 import CopyToClipboard from "@/components/CopyToClipboard"
```

Comment:
@wackerow lmk if you have in mind migrating this component as well. If not, I could add it to the shadcn epic.

Reply:
Adding it to the shadcn epic to track it.